sonosco/common/audio_tools.py

```diff
@@ -8,8 +10,25 @@ def get_duration(file_path):
 def transcode_recording(source, destination, sample_rate):
     subprocess.call([f"sox {source}  -r {sample_rate} -b 16 -c 1 {destination}"], shell=True)
 
+
 def transcode_recordings_an4(raw_path, wav_path, sample_rate):
     subprocess.call([f'sox -t raw -r {sample_rate} -b 16 -e signed-integer -B -c 1 \"{raw_path}\" \"{wav_path}\"'], shell=True)
 
+
 def transcode_recordings_ted3(source, destination, start_time, end_time, sample_rate):
-    subprocess.call([f"sox {source}  -r {sample_rate} -b 16 -c 1 {destination} trim {start_time} ={end_time}"],shell=True)
\ No newline at end of file
+    subprocess.call([f"sox {source}  -r {sample_rate} -b 16 -c 1 {destination} trim {start_time} ={end_time}"],shell=True)
+
+
+def add_noise(audio, std=0.005):
+    noise = np.random.randn(len(audio))
+    data_noise = audio + std * noise
+    return data_noise
+
+
+def shift(audio, n_samples=1600):
+    return np.roll(audio, n_samples)
+
+
+def stretch(audio, rate=1):
+    stretched_audio = librosa.effects.time_stretch(audio, rate)
```

Comment:
Instead of librosa stretch use sox speed effect, it produces much higher quality and, besides stretching, it can also speed up

Reply:
I will change everything to use sox in my next pull request regarding data augmentation. I think it's important to have the current version in because other code is based on this interface.